src/locales/en.json

```diff
@@ -10,5 +10,11 @@
   "help": "Help",
   "logout": "Sign out",
   "error_UnavailableStack": "The stack is unreachable (connection timed-out).",
-  "error_UnauthorizedStack": "Some permissions are missing, the application can't access the requested resource on the stack."
+  "error_UnauthorizedStack": "Some permissions are missing, the application can't access the requested resource on the stack.",
+
+  "Categories": {
```

Comment:
I used a capital for the tree root here like in [photos](https://github.com/cozy/cozy-photos-v3/blob/master/src/locales/en.json) but It was a only a personal preference here.